kotsadm/web/src/scss/components/troubleshoot/SupportBundleAnalysis.scss

```diff
@@ -305,6 +333,83 @@
   transform: translateX(-2px);
 }
 
+.Timeline--wrapper {
+  position: relative;
+  width: 100%;
+  padding: 0 0 0 15px;
```

Comment:
we should write this just as `padding-left` since it's the only one that has a value.